src/vs/workbench/contrib/audioCues/browser/audioCues.contribution.ts

```diff
@@ -0,0 +1,29 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { localize } from 'vs/nls';
+import { Extensions as ConfigurationExtensions, IConfigurationRegistry } from 'vs/platform/configuration/common/configurationRegistry';
+import { Registry } from 'vs/platform/registry/common/platform';
+import { Extensions as WorkbenchExtensions, IWorkbenchContributionsRegistry } from 'vs/workbench/common/contributions';
+import { AudioCueContribution } from 'vs/workbench/contrib/audioCues/browser/audioCueContribution';
+import { LifecyclePhase } from 'vs/workbench/services/lifecycle/common/lifecycle';
+
+Registry.as<IWorkbenchContributionsRegistry>(WorkbenchExtensions.Workbench).registerWorkbenchContribution(AudioCueContribution, LifecyclePhase.Eventually);
+
+Registry.as<IConfigurationRegistry>(ConfigurationExtensions.Configuration).registerConfiguration({
+	'properties': {
+		'audioCues.enabled': {
+			'type': 'string',
+			'description': localize('audioCues.enabled', "Controls whether audio cues are enabled."),
+			'enum': ['smart', 'on', 'off'],
```

Comment:
Not a big fan of using `smart` for a setting. When I change the setting, I am no longer `smart`? Would use `default` instead.

Reply:
Yeah we can call it "auto" or "default" instead. I suggest 'auto"

Reply:
@isidorn I don't mind `smart`, but also find it a little bit misleading. What about `forScreenReaders`?

Reply:
I suggest to go with with auto because we already have
`"editor.accessibilitySupport": "auto" | "on" | "off"`
So we are consistent.
If you feel strongly about `default` that also works. `default` | boolean also works for me.

Reply:
`auto` is fine for me as well